tests/conftest.py

```diff
@@ -35,13 +35,16 @@ def builder(parallel_processing_enabled, tmp_path):
     return builder
 
 
+class ProcessManager(SyncManager):
+    pass
+
+
+ProcessManager.register("ResettingCounter", ResettingCounter)
```

Comment:
Obviously I'm missing something, but why don't we need to register `SynchronizedSet` here?

Reply:
This is a different multiprocessing manager than the one we use in `ExternalProcessLoggingManager`. This one is responsible for sharing test objects between processes. Its' only used in `make_counter` and `make_list` fixtures. 

The purpose here is to not pollute the manager that bionic uses. I also don't want to replace the manager that bionic creates with a test one.  

I guess your other comment in valid that we should rename this to avoid confusion.